static/js/containers/ProfilePage.js

```diff
@@ -42,6 +42,7 @@ class ProfilePage extends React.Component {
     return dispatch(validateProfile(profile, requiredFields, messages)).then(() => {
       dispatch(saveProfile(SETTINGS.username, profile)).then(() => {
         dispatch(clearProfileEdit());
+        return Promise.resolve();
```

Comment:
Now that I think about it, the `return Promise.resolve();` should not be necessary at all. `then` should always return a resolved promise by default, and we know that `dispatch(clearProfileEdit` will never need to return a rejected promise, so it should just work. Need more :coffee: when I talk about these things :)